guides/source/api_documentation_guidelines.md

```diff
@@ -16,7 +16,8 @@ RDoc
 ----
 
 The [Rails API documentation](http://api.rubyonrails.org) is generated with
-[RDoc](http://docs.seattlerb.org/rdoc/).
+[RDoc](http://docs.seattlerb.org/rdoc/). To generate it, make sure you are
+in the `rails` root directory, run `bundle install` and execute:
```

Comment:
Fair enough. I took it from the generation section of the HTML Guides where `guides` is also quoted ;)